controls/SV-245540.rb

```diff
@@ -32,17 +32,23 @@
   tag cci: ['CCI-001233']
   tag nist: ['SI-2 (2)']
 
-  if virtualization.system.eql?('docker')
+  only_if('Control not applicable within a container', impact: 0.0) do
+    !virtualization.system.eql?('docker')
+  end
+
+  if input('skip_endpoint_security_tool')
     impact 0.0
-    describe 'Control not applicable within a container' do
-      skip 'Control not applicable within a container'
+    describe 'We do not want to implement the Endpoint Security for Linux Threat Prevention tool' do
+      skip 'We do not want to implement the Endpoint Security for Linux Threat Prevention tool'
```

Comment:
```suggestion
      skip 'The organization has chosen not to implement the Endpoint Security for Linux Threat Prevention tool'
```

@aaronlippold Should this be not reviewed and then attested to by the organization instead of impact 0.0 since the STIG says it should have it enabled?